config/StoredProcedures/AddCustomer.sql

```diff
@@ -0,0 +1,16 @@
+
+use imp_db_dev;
```

Comment:
Can we please leave these out? We want the API to define which db to use so that it works on both dev and prod